Sources/BottomSheet/BottomSheet+ViewModifiers/BottomSheet+AccountForKeyboardHeight.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BottomSheet+AccountForKeyboardHeight.swift
+//  
+//
+//  Created by Robin Pel on 06/09/2022.
+//
+
+import Foundation
+
+public extension BottomSheet {
+    
+    /// Adds padding to the bottom of the main content when the keyboard appears so all of the main content is visible.
+    ///
+    /// - Note: If the height of the sheet is smaller than the height of the keyboard, this modifier will not make the content visible.
+    ///
+    /// - Parameters:
+    ///   - bool: A boolean whether the option is enabled.
+    ///
+    /// - Returns: A BottomSheet with its main content shifted up to account for the keyboard when it has appeared.
```

Comment:
I make the documentation comment always the same as the text in the readme. So that it is easier to find all available information in both places and that I can change them easily.